### ISSUE TITLE
Open a diagram via drag and drop doesn't work anymore

### ISSUE THREAD

**Issue by RoccoMarco** [jgraph/drawio-desktop#1919]
### Preflight Checklist

* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio-desktop/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Describe the bug**
When dragging and dropping a draw.io diagram onto the draw.io desktop application, it no longer opens the file when dropped on the ribbon bar.

**To Reproduce**
1. Open draw.io desktop.
2. Drag and drop a `.drawio` file onto the drawing area.
   - Result: The diagram is pasted into the existing diagram (this works as expected).
3. Drag and drop a `.drawio` file onto the ribbon bar.
   - Result: The application does not open the diagram as a new file (this does not work).

**Expected behavior**
When a `.drawio` file is dragged onto the ribbon bar, the application should open the file as a new diagram, linked to the dropped file.

**Screenshots**
N/A (if necessary, add screenshots here).

**draw.io version (In the Help->About menu of the draw.io editor):**
- draw.io version: 24.7.17

**Desktop (please complete the following information):**
 - OS: Windows 11 Version 23H2 (OS Build 22631.4460)

**Additional context**
I tried reinstalling the application, but the issue persists. This behavior was observed in version 24.7.17 on Windows.

**Comment by davidjgraph** [comment]
I don't think we ever claimed to have support for dropping into anything other than the editor area.
